src/cron.ts

```diff
@@ -127,13 +151,27 @@ export const cron = async (startDate: Date, owner: string, repo: string, octokit
     for (const [pr, remark] of prRemarks) {
       // We compare the hash to see if there is a match
       const tx = api.tx.system.remark(remark);
-      const match = remarks.find(({ hash }) => hash === tx.method.hash.toHex() || hash === tx.method.toHex());
+
+      // We first start with ongoing referendas
+      const match = ongoing.find(({ hash }) => hash === tx.method.hash.toHex() || hash === tx.method.toHex());
       if (match) {
         logger.info(`Found existing referenda for PR #${pr}`);
         const msg = `Voting for this referenda is **ongoing**.\n\nVote for it [here](${match.url})`;
         rows.push([`${owner}/${repo}#${pr}`, `<a href="${match.url}">${match.url}</a>`]);
         await octokit.rest.issues.createComment({ owner, repo, issue_number: pr, body: msg });
       }
+
+      // if we don't find a match, we search for a closed referenda
+      const closedMatch = completed.find(({ hash }) => hash === tx.method.hash.toHex() || hash === tx.method.toHex());
+      if (closedMatch) {
+        logger.info(`Found completed referenda for PR #${pr}`);
+        const msg = `/rfc process ${closedMatch.executedHash}`;
```

Comment:
so it will request bot to check and merge?
may be better to start from suggesting this command to `copy` and paste by owner? for at least first few weeks, and then if it all work as expected - make it automatic